### DISCUSSION TITLE
Redirect für Hotlinks im CMS

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#191] [Q&A]
In unserem Portalverbund gibt es zwei unterschiedliche Host Adressen für externe und für interne Zugriffe.
Das macht Probleme, wenn wir für Hotlinks in Attributtabellen eine Portaladresse verwenden.
Wir müssten dann in der Hotlink Url (im CMS) zwei unterschiedliche Host Adressen eintragen. Je nachdem von wo aus der Aufruf erfolgt (also entweder über den externen oder den internen Portalzugang).
Daher wäre für uns eine Lösung, wenn man in der Hotlink Url (analog der api.config mit dem redirect) eine Url mit http-header-Variablen eintragen könnte wie:
{header:X-ORIG-SCHEME}://{header:X-ORIG-HOST}/...yada-yada...datalinq/report/gdb@gst@view?kgngstnr=[kg_gst]
Dann würde je nach Aufruf (intern oder extern) der richtige Hostname eingetragen werden.
Ist das möglich?

**Comment by gis-eni** [comment]
Kommt in WebGIS Version >= 6.24.3701
